packages/@aws-cdk/aws-appmesh/test/integ.mesh.expected.json

```diff
@@ -599,6 +599,49 @@
         }
       }
     },
+    "meshrouterroute417D3F5B5": {
+      "Type": "AWS::AppMesh::Route",
+      "Properties": {
+        "MeshName": {
+          "Fn::GetAtt": [
+            "meshACDFE68E",
+            "MeshName"
+          ]
+        },
+        "RouteName": "route-4",
+        "Spec": {
+          "GrpcRoute": {
+            "Action": {
+              "WeightedTargets": [
+                {
+                  "VirtualNode": {
+                    "Fn::GetAtt": [
+                      "meshnode3D2A19CF2",
+                      "VirtualNodeName"
+                    ]
+                  },
+                  "Weight": 20
+                }
+              ]
+            },
+            "Match": {
+              "ServiceName": {
+                "Fn::GetAtt": [
+                  "meshserviceE06ECED5",
+                  "VirtualServiceName"
+                ]
+              }
+            }
+          }
+        },
+        "VirtualRouterName": {
+          "Fn::GetAtt": [
+            "meshrouter81B8087E",
+            "VirtualRouterName"
+          ]
+        }
+      }
+    },
```

Comment:
Hmm, why changes to this file? I understand the API changed, but after updating the code to use it, the resulting template should be the same as before...

Reply:
Added the other route types to the integ file. Removing them